test/unit/ssz_ex_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule Unit.SSZExTest do
+  alias LambdaEthereumConsensus.SszEx
+  use ExUnit.Case
+
+  def assert_roundtrip(hex_serialized, deserialized, schema) do
+    serialized = Base.decode16!(hex_serialized)
+    assert ^serialized = SszEx.encode(deserialized, schema)
+    assert deserialized === SszEx.decode(serialized, schema)
+  end
+
+  test "serialize and deserialize uint" do
+    assert_roundtrip("05", 5, {:int, 8})
```

Comment:
Instead of using hex, why don't we use the binaries directly? For example:

```elixir
<<5>>
<<5, 0>>
<<5, 0, 0, 0>>
<<5, 0, 0, 0, 0, 0, 0, 0>>
```

Or for a value like 276:

```elixir
<<14, 1>>
<<14, 1, 0, 0>>
<<14, 1, 0, 0, 0, 0, 0, 0>>
```